modules/trustxBidAdapter.md

```diff
@@ -0,0 +1,39 @@
+# Overview
+
+Module Name: TrustX Bidder Adapter
+Module Type: Bidder Adapter
+Maintainer: paul@trustx.org
+
+# Description
+
+Module that connects to TrustX demand source to fetch bids.
+
+# Test Parameters
+```
+    window.globalPrebidTrustxPriceType = 'gross'; // by default is 'net'
```

Comment:
After update you don't need this in markdown. Update params as required.

Reply:
Are you sure you're calling the right prebid location on the page example you've provided? On the page it is "../../build/dev/prebid.js". So I can't see any calls to TX bidder going at all.